tests/backup/backup_restore_basic_test.go

```diff
@@ -595,16 +607,96 @@ var _ = Describe("{ScheduleBackupCreationAllNS}", func() {
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Creation and Validation of schedule backup with schedule name [%s]", backupName))
+			err = suspendBackupSchedule(backupName, periodicPolicyName, orgID, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Suspending Backup Schedule [%s]", backupName))
 			scheduleNames = append(scheduleNames, backupName)
+			backupNames = append(backupNames, firstScheduleBackupName)
 		})
 
 		Step("Restoring scheduled backups", func() {
 			log.InfoD("Restoring scheduled backups")
 			ctx, err := backup.GetAdminCtxFromSecret()
 			log.FailOnError(err, "Fetching px-central-admin ctx")
+			appContextsExpectedInBackup := FilterAppContextsByNamespace(scheduledAppContexts, bkpNamespaces)
 			restoreName = fmt.Sprintf("%s-%s", restoreNamePrefix, firstScheduleBackupName)
-			err = CreateRestore(restoreName, firstScheduleBackupName, namespaceMapping, destinationClusterName, orgID, ctx, nil)
-			dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of restoring scheduled backups - %s", restoreName))
+			err = CreateRestoreWithValidation(ctx, restoreName, firstScheduleBackupName, namespaceMapping, make(map[string]string), destinationClusterName, orgID, appContextsExpectedInBackup)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of restore %s of backup %s", restoreName, firstScheduleBackupName))
+			restoreNames = append(restoreNames, restoreName)
+		})
+
+		Step(fmt.Sprintf("Take manual backup of applications"), func() {
+			log.InfoD(fmt.Sprintf("Taking manual backup of applications"))
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			manualBackupName = fmt.Sprintf("%s-manual-%s", BackupNamePrefix, RandomString(4))
+			appContextsToBackup := FilterAppContextsByNamespace(scheduledAppContexts, bkpNamespaces)
+			err = CreateBackupWithValidation(ctx, manualBackupName, SourceClusterName, backupLocationName, backupLocationUID, appContextsToBackup, labelSelectors, orgID, srcClusterUid, "", "", "", "")
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation and validation of backup [%s] of namespace (scheduled Context) [%s]", manualBackupName, bkpNamespaces))
+			backupNames = append(backupNames, manualBackupName)
+		})
+
+		Step(fmt.Sprintf("Perform default restore of applications by replacing the existing resources"), func() {
+			log.InfoD(fmt.Sprintf("Perform default restore of applications by replacing the existing resources"))
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			defaultRestoreName = fmt.Sprintf("%s-%s", restoreNamePrefix, manualBackupName)
+			err = CreateRestoreWithReplacePolicy(defaultRestoreName, manualBackupName, make(map[string]string), SourceClusterName, orgID, ctx, make(map[string]string), 2)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Creating default restore for manual backup with replace policy [%s]", defaultRestoreName))
```

Comment:
Here can we separately validate the restore using "ValidateRestore"
Refer : Testcase : KubevirtVMBackupRestoreWithDifferentStates

Reply:
Added the restore verification.